packages/world/mud.config.ts

```diff
@@ -330,10 +322,11 @@ export default defineWorld({
       },
       key: ["entityId"],
     },
-    ForceField: {
+    Machine: {
       schema: {
         entityId: "EntityId",
         createdAt: "uint128",
+        accDepletedTime: "uint128",
```

Comment:
what's "acc" here?

Reply:
accumulated, can change to that if you prefer

Reply:
generally find these abbreviations hard to follow, so would prefer the whole word

but would `depletedTime` not be descriptive enough? or `totalDepletedTime`?

Reply:
accumulated made sense to me because it accumulates the different periods where it was depleted, `depletedTime` sounds like "time since it got depleted" imo. `total` works I guess

Reply:
we seem to use `At` suffix for timestamps so `Time` or `Seconds` or something would make sense to me, as long as we're consistent with it

Reply:
Sounds good, yeah we need to do a pass to improve consistency for sure